Examples/Conference/App/Shared/Screens/Conference/ConferenceViewModel.swift

```diff
@@ -164,10 +164,10 @@ extension ConferenceViewModel {
         Task { @MainActor in
             do {
                 state = .connecting
-                try mediaConnection.setMainAudioTrack(mainLocalAudioTrack)
-                try mediaConnection.setMainVideoTrack(cameraVideoTrack)
-                try mediaConnection.receiveMainRemoteAudio(true)
-                try mediaConnection.receiveMainRemoteVideo(true)
+                try await mediaConnection.setMainAudioTrack(mainLocalAudioTrack)
+                try await mediaConnection.setMainVideoTrack(cameraVideoTrack)
+                try await mediaConnection.receiveMainRemoteAudio(true)
+                try await mediaConnection.receiveMainRemoteVideo(true)
```

Comment:
Hm. Why do you want to await these? Arent those just fire&forget basically?

Reply:
Those functions became async because now I'm using Swift actor under the hood, mainly to synchronise access to peer connection and other mutable states. We can of course keep them synchronous or fire&forget like you said, but then we'll loose error handling, which is arguably fine in this case (the only possible error that can be thrown here is when WebRTC transceiver fails to change direction).